locales/ta_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -86,7 +86,7 @@ msgstr ""
 
 msgctxt "install-duckduckgo"
 msgid "%sClick %sAllow%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
-msgstr ""
+msgstr "%sஅழுத்தவும் %அனுமதிக்கவும்%அழுத்தவும் %சேர்க்கவும்%சரிபார்த்துக்கொள்ளவும்%s, பிறகு ஓகேவை அழுத்தவும்%s"
```

Comment:
It looks like a the placeholder tags got messed in here. Would you mind checking it again? I include a possible suggestion below, but I don't know the language so you'd better double check.

```suggestion
msgstr "%sஅழுத்தவும் %sஅனுமதிக்கவும்%sஅழுத்தவும் %sசேர்க்கவும்%sசரிபார்த்துக்கொள்ளவும்%s, பிறகு %sஓகேவை%s அழுத்தவும்"
```